pkgs/development/tools/locate-dominating-file/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  bats,
+  bash,
+  lib,
+  stdenv,
```

Comment:
Given that your package is a Bash shell script, you can use `stdenvNoCC` - a stdenv without C compiler.